Revert-and-Regression-Policy.md

```diff
@@ -56,17 +64,20 @@ You should also begin debugging your changes to figure out what caused the regre
 
 Once the dev workflow leads decide whether to revert, fix forward, or both, you should start work on the associated actions (re-creating the PR with the bug fixed, opening a fix-forward PR to fix the bug, or both).
 
+### Developers with Revert Permissions
+
+Any Oppia developer with permission to open revert PRs on GitHub may decide, following the above policy, whether to revert the PR, fix forward, pursue both in parallel, or make one of the two allowed exceptions. They should document their reasoning and communicate on the tracking issue so the dev workflow leads can understand what happened when they consider mitigations.
```

Comment:
The dev workflow leads are responsible for considering mitigations to prevent a similar regression from happening again. This is discussed in the dev workflow leads section. I'll remove that bit though since it's not necessary